skimage/transform/tests/test_radon_transform.py

```diff
@@ -80,21 +81,31 @@ def check_radon_center(shape, circle):
 
 shapes_for_test_radon_center = [(16, 16), (17, 17)]
 circles_for_test_radon_center = [False, True]
+dtype_for_test_radon_center = [np.float64, np.float32, np.uint8, bool]
+preserve_range_for_test_radon_center = [False, True]
 
 
-@testing.parametrize("shape, circle",
+@testing.parametrize("shape, circle, dtype, preserve_range",
                      itertools.product(shapes_for_test_radon_center,
-                                       circles_for_test_radon_center))
-def test_radon_center(shape, circle):
-    check_radon_center(shape, circle)
+                                       circles_for_test_radon_center,
+                                       dtype_for_test_radon_center,
+                                       preserve_range_for_test_radon_center))
+def test_radon_center(shape, circle, dtype, preserve_range):
+    check_radon_center(shape, circle, dtype, preserve_range)
 
 
 rectangular_shapes = [(32, 16), (33, 17)]
+dtype_for_test_radon_center = [np.float64, np.float32, np.uint8, bool]
+preserve_range_for_test_radon_center = [True, False]
```

Comment:
once you stop using itertools product, these definitions are likely useless too.